releasenotes/notes/35293.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+issue:
+- 35068
+releaseNotes:
+- |
+  **Fixed** An issue when service entry and gateway created at same time, the service entry is ignored some times.
```

Comment:
Maybe **Fixed** an issue when creating a ServiceEntry and Gateway at the same time where the ServiceEntry is occasionally not created.